tsl/test/expected/continuous_aggs_invalidation.out

```diff
@@ -178,7 +189,7 @@ SELECT * FROM cagg_invals;
 (4 rows)
 
 -- Refreshing below the threshold does not move it:
-CALL refresh_continuous_aggregate('cond_10', 20, 49);
+CALL refresh_continuous_aggregate('cond_10', 0, 49);
```

Comment:
why do we have to change this call? IIUC The existing call wouldn't affect the invalidations or threshold. Am I missing something here?

Reply:
It is leftover before I added call for `0, 50` above. I will fix this back.

Reply:
Fixed